### PR TITLE
JENKINS-67071 Adjust class for section header change

### PR DESCRIPTION
**Pull request by timja**
Bug in https://github.com/jenkinsci/jenkins/pull/5842 which is unreleased

see https://issues.jenkins.io/browse/JENKINS-67071

cc @janfaracik

### CODE REVIEW

**Review by uhafner — APPROVED** [review]
Clicking the sections does work again!

**Review by fqueiruga — APPROVED** [review]

**Comment by timja** [comment]
This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Pull request merged**
